definitions.js

```diff
@@ -189,12 +225,37 @@ export const createModelDefinition = definition => {
     `;
   const toGraphQL = () =>
     v(`    
-        ${toGraphQLType(definition)}
-        ${toGraphQLInput(definition)}
-        ${toGraphQLFragment(definition)}
-        ${toGraphQLQueries(definition)}
-        ${toGraphQLMutations(definition)}
-  `);
+      input PaginationActionInput {
```

Comment:
Did you check what will happen with two types? I believe this declaration will produce an error because it's going to be duplicated. Also all the others.

I believe it's possible to ignore this error on GraphQL options but it would be better to avoid re-declarations.

Reply:
I see, is there a known way of declaring this only once for all definitions? I'll look into it.